desktop/release-notes.md

```diff
@@ -33,6 +33,60 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/ge
 
```

Comment:
the {% include desktop-install.html %} also needs to be updated as does the Linux download links for each linux distribution